### PR TITLE
added support for resource group name and doc fix

### PR DESCRIPTION
**Pull request by ujjwal-ibm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,56 +176,60 @@ The top-level `source` block defines reusable builder configuration blocks.
...
+image_name | string | Optional | The name of the resulting custom image that will appear in your account. Required.
+encryption_key_crn | string | Optional | The CRN of the [Key Protect Root Key](https://cloud.ibm.com/docs/key-protect?topic=key-protect-getting-started-tutorial) or [Hyper Protect Crypto Services Root Key](https://cloud.ibm.com/docs/hs-crypto?topic=hs-crypto-get-started) for this resource.
+communicator | string | Required | Communicators are the mechanism Packer uses to upload files, execute scripts, etc. with the machine being created. Choose between "ssh" (for Linux) and "winrm" (for Windows). Required.
+tags | list | Optional | List of user tags for this image. Tags can be made as `key:value` pair or in `label` format.
 ***Linux Communicator Variables*** |
-ssh_username | string | The username to connect to SSH with.
-ssh_port | int |The port that SSH will be available on. Defaults to port 22.
-ssh_timeout | string | The time to wait for SSH to become available before timing out. The format of this value is a duration such as "5s" or "5m".
+ssh_username | string | Optional | The username to connect to SSH with.
```
Comment (astha-jain) [inline README.md:222]: should be required?

File: developer/examples/build.vpc.centos-resource-group-name.pkr.hcl [hunk 2]
```diff
@@ -42,30 +43,7 @@ source "ibmcloud-vpc" "centos" {
...
-  ssh_timeout  = "15m"
-
-  timeout = "30m"
-}
-
-source "ibmcloud-vpc" "centos-other" {
-  api_key           = var.IBM_API_KEY
-  region            = var.REGION
-  subnet_id         = var.SUBNET_ID
-  resource_group_id = var.RESOURCE_GROUP_ID
```
Comment (astha-jain) [inline developer/examples/build.vpc.centos-resource-group-name.pkr.hcl:68]: why are we removing this?

File: README.md (hunk 1 above)
Comment (ujjwal-ibm) [reply README.md]: it defaults to root, updated the readme

File: developer/examples/build.vpc.centos-resource-group-name.pkr.hcl (hunk 2 above)
Comment (ujjwal-ibm) [reply developer/examples/build.vpc.centos-resource-group-name.pkr.hcl:68]: was a redundant file

**Review by astha-jain — APPROVED** [review]

**Pull request merged**